ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource

```diff
@@ -246,27 +246,31 @@ Click CheckBox Single Model Serving Platforms
     ${status}=  Get Checkbox State Of Single Modelserving platforms
     IF  "${status}"=="False"
         Select Checkbox   ${SINGLE_MODEL_SERVING_CHECK_BOX}
+        Save Changes In Cluster Settings
     END
 
 Click CheckBox Multi Model Serving Platforms
     [Documentation]    Enables Multi Modelserving platforms
     ${status}=  Get Checkbox State Of Multi Modelserving platforms
     IF  "${status}"=="False"
         Select Checkbox   ${MULTI_MODEL_SERVING_CHECK_BOX}
+        Save Changes In Cluster Settings
     END
```

Comment:
by looking at how the keywords are used in the test case, this keyword should be called more like `Select Multi Model Serving ...` because `Click` can either means select or deselect

Reply:
done can u plz take a look